### ISSUE TITLE
Chart2D control with 'FastLine' option throws System.OverflowException: Overflow error for big differences between chart area values

### ISSUE THREAD

**Issue by PawelTroka** [PawelTroka/Computator.NET#5]
Reproduction
Input:
FastLine; f(x)=x; x0=-5·10⁻⁷; xN=5·10⁷; y0=-3·10⁻⁷; yN=3·10⁻⁷

With FastLine for some parameters, probably because of the fact that GDI+ has hard bounds for drawing coordinates for DrawLine method.

```
    System.OverflowException: Overflow error.
       at System.Drawing.Graphics.CheckErrorStatus(Int32 status)
       at System.Drawing.Graphics.DrawLine(Pen pen, Single x1, Single y1, Single x2, Single y2)
       at System.Windows.Forms.DataVisualization.Charting.ChartTypes.FastLineChart.DrawLine
```

**Comment by PawelTroka** [comment]
Half resolved by not using FastLine with commit 775b49c71278c9e1c84164ec5eaab53bcff04b86.

**Comment by PawelTroka** [comment]
Since we don't have any problems with it recently and we are not using FastLine option anymore, I am closing this one.
